api.json

```diff
@@ -125,6 +125,26 @@
             "description": "Safari: Password for safari_apns_p12 file",
             "writeOnly": true
           },
+            "apns_key_id": {
+              "type": "string",
+              "description": "iOS: Required if using p8. Unique identifier for the p8 authentication key.",
+              "nullable": true
```

Comment:
I think we don't need nullable anymore. All the properties in OpenAPI spec should be optional by default.

Reply:
Fixed.